docs/docs/guides/how-to-create-syncs.mdx

```diff
@@ -0,0 +1,99 @@
+import ThreeDotMenu from './assets/sync-with-google-sheets-three-dot-menu.png';
+import CreateNewButtonModal from './assets/sync-with-google-sheets-create-modal.png';
+import CreateNewModal from './assets/sync-with-google-sheets-create-new-modal.png';
+import CreateNewModalFilled from './assets/sync-with-google-sheets-create-new-modal-filled.png';
+import EditDeleteSync from './assets/sync-with-google-sheets-edit-delete-options.png';
+import DashboardOverview from './assets/sync-with-google-sheets-dashboard-overview.png';
+import DashboardOverviewThreeDotMenu from './assets/sync-with-google-sheets-dashboard-overview-three-dot-menu.png';
+
+# Syncs
+
+Syncs let you send your chart results to a Google Sheet in your Google Drive on a schedule. You can set up as many syncs as you like, and if you make any changes to the content, Lightdash will update the syncs the next time they’re sent.
+
+## Enabling scheduled deliveries
+
+If you are running Lightdash on self-host, make sure you have set up Google Sheets integration before you start. See [setting up Google Sheets](self-host/customize-deployment/google-sheets-integration.mdx) for more information.
+If you're not on a self-hosted instance (e.g. Cloud or Enterprise), then your instance comes preconfigured with Google Sheets integration, so you don't need to do anything to enable it.
```

Comment:
There should be a line break before this sentence. 

> If you're using Lightdash Cloud, this is already set up and available for you to use immediately.

Reply:
done